test.py

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env python3
```

Comment:
`test.py -h` or `test-py --help` would be a nice feature, independent of the instructions in the documentation of the project.